frontend/src/components/GoalForm/Form.js

```diff
@@ -127,6 +128,40 @@ export default function Form({
 
       <h3 className="margin-top-4 margin-bottom-3">Goal summary</h3>
 
+      {collaborators.length && collaborators.map((collaborator) => {
+        const {
+          goalCreatorName,
+          goalCreatorRoles,
+          goalNumber,
+        } = collaborator;
+        if (!goalCreatorName) return null;
+        return (
+          <FormGroup key={goalNumber}>
+            <Label htmlFor="entered-by" className="text-bold">
```

Comment:
Can we switch to use ```<ReadOnlyField />```? We shouldn't have a label that doesn't point to a form control (I know there are random ones floating around, trying to rein it in)